pom.xml

```diff
@@ -110,8 +113,11 @@
 					<excludes>
 						<exclude>**/generated/**</exclude>
 						<exclude>**/se/sundsvall/webmessagecollector/Application*</exclude>
-						<exclude>**/se/sundsvall/webmessagecollector/integration/db/model/**/*_.*</exclude>
-						<exclude>**/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.*</exclude>
+						<exclude>**/se/sundsvall/webmessagecollector/integration/db/model/**/*_.*
+						</exclude>
+						<exclude>
+							**/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.*
```

Comment:
Why are these files manually coded? Isn't it possible to  generate them?

(If we still want to "own" the source code for the client model, we should at least have tests for them, i.m.h.o)